src/background.js

```diff
@@ -6,6 +6,8 @@ import "webextension-polyfill";
 
 import { Rally, runStates } from "@mozilla/rally";
 
+import * as webScience from "@mozilla/web-science";
```

Comment:
@jonathanmayer this causes a lint failure:
`error  'webScience' is defined but never used  no-unused-vars`

Can you make it do something more interesting?